source/scss/components/_rating-stars.scss

```diff
@@ -1,42 +1,30 @@
 // HOTEL CATEGORY IMAGE ELEMENTS (previously known as Stars)
 
 .mot_stars {
-  display: flex;
-  padding-top: 2px;
+  display: inline-flex;
   white-space: nowrap;
-  width: 72px;
 
   .rating-star:first-child {
     margin-left: 0;
   }
 }
 
 .rating-star {
-  background-color: $orange;
-  border: 1px solid $orange;
+  background-color: $font-color;
+  border: 1px solid $font-color;
   display: inline-block;
-  height: 12px;
-  margin-left: 3px;
+  height: $p-font-size / 2;
+  margin-left: 2px;
```

Comment:
I still believe this value shouldn't be fixed, especially on larger font sizes. There must be a way of calculating it; I think we just didn't find it yesterday.

Reply:
It is fixed to p-font-size anyway, because CSS is not dynamic. If you zoom, the font-size will grow as well, as everything will grow. 
We should have a follow up for this, if we really need it, but then connected to find a proper solution making the height and width dependent on the parent's font-size with Javascript.

Reply:
Yeah, it's not necessary if the editors never add the starts to texts with very large fonts.